CHANGELOG.md

```diff
@@ -58,6 +58,7 @@ There are breaking changes in this release:
 - [#4681](https://github.com/influxdb/influxdb/pull/4681): Increase default buffer size for collectd and graphite listeners
 - [#4659](https://github.com/influxdb/influxdb/pull/4659): Support IF EXISTS for DROP DATABASE
 - [#4685](https://github.com/influxdb/influxdb/pull/4685): Automatically promote node to raft peer if drop server results in removing a raft peer.
+- [#4846](https://github.com/influxdb/influxdb/pull/4846): Allow NaN as a valid value on the graphite service; discard these points silently (graphite compatibilty).
```

Comment:
Be sure to give yourself credit here, by thanking yourself. :-) Add "thanks @jsternberg" to the end of this line.